README.md

```diff
@@ -39,7 +39,11 @@ limitations under the License.
 </p>
 
 <h4 align="center">
-    <p><b>English</b> | <a href="https://github.com/huggingface/transformers/blob/master/README_zh-hans.md">简体中文</a><p>
+    <p>
+        <b>English</b> |
+        <a href="https://github.com/huggingface/transformers/blob/master/README_zh-hans.md">简体中文</a> |
+        <a href="https://github.com/huggingface/transformers/blob/master/README_zh-tw.md">繁體中文</a>
```

Comment:
I recommend `zh-hans` and `zh-hant` instead of `zh-cn` and `zh-tw` to be more inclusive.

Reply:
Sure, that makes sense.